src/MGroup.MSolve.AnalysisWorkflow/IChildAnalyzer.cs

```diff
@@ -6,5 +10,6 @@ public interface IChildAnalyzer: IAnalyzer
 		//      parent analyzer to the child analyzers constructor, before the parent analyzer constructor ends. However that is
 		//      too much trouble without a specific need for that degree of immutability.
 		IParentAnalyzer ParentAnalyzer { get; set; }
+		Dictionary<int, IVector> TotalSolution { get; set; }
```

Comment:
Please change name to Responses - we are considering the stimulus/response wording